docs/release/trg-2/trg-2-5.md

```diff
@@ -2,23 +2,27 @@
 title: TRG 2.05 - Repository metafile
 ---
 
-| Status     | Created     | Post-History       |
-|------------|-------------|--------------------|
+| Status     | Created     | Post-History           |
+|------------|-------------|------------------------|
+| Update     | 10-Jan-2024 | reword the why section |
 | Active     | 24-Apr-2023 |
-| Prerelease | 7-Mar-2023  | Moved out of draft |
-| Draft      | 24-Feb-2023 | Draft release      |
+| Prerelease | 7-Mar-2023  | Moved out of draft     |
+| Draft      | 24-Feb-2023 | Draft release          |
 
 ## Why
 
-Due to many products having more than one repository, we need a way of identifying dependent repositories automatically. This allows us to analyze repositories for quality checks and for generating documentation across repositories.
+Many products are build from more than one repository.
+We aim to have automatic quality checks and therefore need a metadata file to identify the repositories belonging to a product.
 
 ## Description
 
-To enable automatic analysis of repositories, all repositories **MUST** have a file named `.tractusx` present on root-level.
+To enable automatic analysis of repositories, all repositories **MUST** have a file named `.tractusx` present on root-level with the dedicated `repoCategory` which points to the displayed section for the repository.
 This file needs to contain metadata about the repository and product in a YAML format.
 The mandatory declarations in the `.tractusx` metadata file do vary for leading- and non-leading repositories.
 
-Documentation on the content of the `.tractusx` metadata file can be found [in the quality check docs](https://github.com/eclipse-tractusx/tractusx-quality-checks/blob/main/docs/metadata_file.md).
+Detailed documentation on the content of the `.tractusx` metadata file can be found [in the quality check docs](https://github.com/eclipse-tractusx/tractusx-quality-checks/blob/main/docs/metadata_file.md).
```

Comment:
I would stop pointing to old quality checks repo and replace it with a link to sig-release instead:

https://github.com/eclipse-tractusx/sig-release/blob/main/docs/metadata_file.md

Reply:
[1831649](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/581/commits/18316494d3afbaf6d8c6d92d36bcac785ceba4d9)